SPECS/reaper/reaper.spec

```diff
@@ -184,7 +185,7 @@ fi
 
 %changelog
 * Mon Feb 17 2025 Kanishk Bansal <kanbansal@microsoft.com> - 3.1.1-16
```

Comment:
Did this package not publish before, or are we missing a release bump here?

Reply:
Good eyes...but in this case, it's the right change.  I took half-the-fix on fasttrack and then discovered a second fix was waiting in the wings.  We only published the combined one.